Resources/Prototypes/_DV/Entities/Objects/Weapons/Melee/home_run_bat.yml

```diff
@@ -14,7 +14,13 @@
     soundHit:
       collection: ExplosionSmall
   - type: MeleeRequiresWield # You can't hit a home run with one hand, jimbo.
+  - type: IncreaseDamageOnWield
+    damage:
+      types:
+        Blunt: 10
+        Structural: 25 # Get on WHACKIN
```

Comment:
We have MeleeRequiresWield already, so just increase the damage on the MeleeWeapon component and remove IncreaseDamageOnWield